taxi/tests/test_forms.py

```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+
+
+class FormsTest(TestCase):
+    def test_driver_creation_form_with_license_first_last_name_is_valid(self):
+        form_data = {
+            "username": "new_user",
+            "password1": "user12test",
+            "password2": "user12test",
+            "first_name": "Test first",
+            "last_name": "Test last",
+            "license_number": "HFJ12341",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)

```

Comment:
The assertion `self.assertEqual(form.cleaned_data, form_data)` might not work as expected if the form modifies the data during cleaning (e.g., password hashing). Consider checking only the relevant fields or using `assertDictContainsSubset` to avoid potential failures.